src/lib/y2partitioner/actions/controllers/encryption.rb

```diff
@@ -69,16 +72,19 @@ def show_dialog?
         #
         # @see #action
         #
-        # If there is more than one possible action, the user should be able to
-        # use the UI to select which one to perform
+        # If there is more than one possible action, the user should be able to use the UI to select
+        # which one to perform. Note that :sanitize and :remove are not directly offered to the user in
+        # the UI.
         #
         # @return [Array<Symbol>]
         def actions
           return @actions if @actions
 
           @actions =
             if fs_controller.encrypt
-              if can_keep?
+              if sanitize_encryption?
+                [:sanitize, :encrypt]
```

Comment:
I find this part quite confusing. In `[:keep, :encrypt]` it's obvious we are offering both options to the user. But here that's not the case... Even if I would manage to understand it now, looks like one of those things that nobody will see clearly in the future.

Reply:
Yes, similar to *:remove* action. There is a clear translation between actions and what the user can select. Do you propose another approach?

Reply:
I see actions as things that can be done over an encryption. Sometimes, the actions are selected by the user, but other times they are automatically selected.

Reply:
As I can see, this works because the widget contains logic to enforce the `encrypt` option... which again makes this a little bit... unobvious